hal/src/main/native/athena/HAL.cpp

```diff
@@ -256,6 +257,28 @@ uint64_t HAL_GetFPGATime(int32_t* status) {
   return (upper2 << 32) + lower;
 }
 
+uint64_t HAL_ExpandFPGATime(uint32_t unexpanded_lower, int32_t* status) {
```

Comment:
Does there need to be a simulation version of this function?

Reply:
I added it. Nothing would really use it though.